web/src/components/LandingPageSlider.tsx

```diff
@@ -0,0 +1,154 @@
+import { QUERIES, ROUTES, routeWithQuery } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement, useRef, useState } from "react";
+import Slider, { CustomArrowProps } from "react-slick";
+import { useMountEffect } from "../lib/utils/helpers";
+
+function NextArrow(props: CustomArrowProps) {
+  return (
+    <button className="slider-button next" onClick={props.onClick} aria-hidden="false">
+      <img src="/img/nextArrowIcon.svg" alt="next-arrow" />
+    </button>
+  );
+}
+function PrevArrow(props: CustomArrowProps) {
+  return (
+    <button className="slider-button prev" onClick={props.onClick} aria-hidden="false">
+      <img src="/img/prevArrowIcon.svg" alt="previous-arrow" />
+    </button>
+  );
+}
+
+export const LandingPageSlider = (): ReactElement => {
+  const router = useRouter();
+  const { Config } = useConfig();
+  const [activeTile, setActiveTile] = useState(0);
+  const [slideBreakPoint, setSlideBreakPoint] = useState(7);
+  const cardRef = useRef<null | HTMLButtonElement>(null);
+  const sliderRef = useRef<null | HTMLDivElement>(null);
+
+  const getNumberOfTilesToScroll = (): number => {
+    const sliderWidth = sliderRef.current?.clientWidth;
+    const cardWidth = cardRef.current?.clientWidth;
+    const numberOfTiles = 7;
+
+    if (!sliderWidth || !cardWidth) return numberOfTiles;
+    const widthAdjustment = 100;
+    const numberOfTilesToFit = Math.ceil(sliderWidth / (cardWidth + widthAdjustment));
+    const numberOfTilesToScroll = numberOfTiles - numberOfTilesToFit;
+    return numberOfTilesToScroll;
+  };
+
+  useMountEffect(() => {
+    setSlideBreakPoint(getNumberOfTilesToScroll());
+    function handleResize() {
+      setSlideBreakPoint(getNumberOfTilesToScroll());
+    }
+    window.addEventListener("resize", handleResize);
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  });
+
+  const routeToOnboarding = () => {
+    router.push(ROUTES.onboarding);
+    analytics.event.landing_page_hero_get_started.click.go_to_onboarding();
+  };
+
+  const setFlowAndRouteUser = (flow: "starting" | "out-of-state" | "up-and-running") => {
+    routeWithQuery(router, {
+      path: ROUTES.onboarding,
+      queries: { [QUERIES.flow]: flow },
+    });
+  };
+
+  const settings = {
+    className: "slider variable-width",
+    slidesToShow: 1,
+    slidesToScroll: 2,
+    variableWidth: true,
+    dots: false,
+    arrows: true,
+    infinite: false,
+    speed: 500,
+    nextArrow: activeTile < slideBreakPoint ? <NextArrow /> : undefined,
+    beforeChange: (previous: number, next: number) => {
+      setActiveTile(next);
+    },
+    prevArrow: activeTile > 0 ? <PrevArrow /> : undefined,
+  };
+
+  return (
+    <div className="slider" ref={sliderRef} style={{ marginLeft: "25px", marginRight: "25px" }}>
+      <Slider {...settings}>
+        <LandingPageActionTile
+          // reference={cardRef}
```

Comment:
commented code should be removed... but why is this commented? don't you need this ref?